.github/dependabot.yml

```diff
@@ -1,16 +1,11 @@
 version: 2
 updates:
-  - package-ecosystem: 'github-actions'
```

Comment:
I don't know if we want to reinstate this, probably not - the view of the stewards seems to be to not pull in arbitrary github actions versions, instead to lock them down to individual commits managed by Unified CI, apart from a few blessed actions.